README.md

```diff
@@ -77,30 +77,30 @@ latest crate code is, the crate can be in one of 3 states:
 
 These are the empty repositories that have PRs waiting to be merged.
 
-- [linux-loader](https://github.com/rust-vmm/linux-loader): parser and loader
-  for vmlinux and bzImage images as well as some other helpers for kernel
-  commandline.
-- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
-  bindings to virtio kernel headers generated using
-  [bindgen](https://crates.io/crates/bindgen).
 - [vmm-vcpu](https://github.com/rust-vmm/vmm-vcpu/): a hypervisor-agnostic
   abstraction for Virtual CPUs (vCPUs).
 
 ### rust-vmm
 
-- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
-  virtual machine's memory.
 - [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
```

Comment:
We should also add event-manager here: https://github.com/rust-vmm/event-manager